src/BaseGraph.ts

```diff
@@ -0,0 +1,956 @@
+   * @returns {Promise<MicrosoftGraph.Person[]>}
+   * @memberof IGraph
+   */
+  findPerson(query: string): Promise<MicrosoftGraph.Person[]>;
+
+  /**
+   * async promise, returns a Graph contact associated with the email provided
+   *
+   * @param {string} email
+   * @returns {Promise<MicrosoftGraph.Contact[]>}
+   * @memberof IGraph
+   */
+  findContactByEmail(email: string): Promise<MicrosoftGraph.Contact[]>;
+
+  /**
+   * async promise, returns Graph contact and/or Person associated with the email provided
+   * Uses: Graph.findPerson(email) and Graph.findContactByEmail(email)
+   *
+   * @param {string} email
+   * @returns {(Promise<Array<MicrosoftGraph.Person | MicrosoftGraph.Contact>>)}
+   * @memberof IGraph
+   */
+  findUserByEmail(email: string): Promise<Array<MicrosoftGraph.Person | MicrosoftGraph.Contact>>;
+
+  /**
+   * async promise, returns Graph photo associated with the logged in user
+   *
+   * @returns {Promise<string>}
+   * @memberof IGraph
+   */
+  myPhoto(): Promise<string>;
+
+  /**
+   * async promise, returns Graph photo associated with provided userId
+   *
+   * @param {string} userId
+   * @returns {Promise<string>}
+   * @memberof IGraph
+   */
+  getUserPhoto(userId: string): Promise<string>;
+
+  /**
+   * async promise, returns Graph photos associated with contacts of the logged in user
+   *
+   * @param {string} contactId
+   * @returns {Promise<string>}
+   * @memberof IGraph
+   */
+  getContactPhoto(contactId: string): Promise<string>;
+
+  /**
+   * async promise, returns Calender events associated with either the logged in user or a specific groupId
+   *
+   * @param {Date} startDateTime
+   * @param {Date} endDateTime
+   * @param {string} [groupId]
+   * @returns {Promise<MicrosoftGraph.Event[]>}
+   * @memberof IGraph
+   */
+  getEvents(startDateTime: Date, endDateTime: Date, groupId?: string): Promise<MicrosoftGraph.Event[]>;
+
+  /**
+   * async promise to the Graph for People, by default, it will request the most frequent contacts for the signed in user.
+   *
+   * @returns {Promise<MicrosoftGraph.Person[]>}
+   * @memberof IGraph
+   */
+  getPeople(): Promise<MicrosoftGraph.Person[]>;
+
+  /**
+   * async promise to the Graph for People, defined by a group id
+   *
+   * @param {string} groupId
+   * @returns {Promise<MicrosoftGraph.Person[]>}
+   * @memberof IGraph
+   */
+  getPeopleFromGroup(groupId: string): Promise<MicrosoftGraph.Person[]>;
+
+  /**
+   * async promise, returns all planner plans associated with the group id
+   *
+   * @param {string} groupId
+   * @returns {Promise<MicrosoftGraph.PlannerPlan[]>}
+   * @memberof IGraph
+   */
+  getPlansForGroup(groupId: string): Promise<MicrosoftGraph.PlannerPlan[]>;
+
+  /**
+   *  async promise, returns all planner plans associated with the user logged in
+   *
+   * @returns {Promise<MicrosoftGraph.PlannerPlan[]>}
+   * @memberof IGraph
+   */
+  planner_getAllMyPlans(): Promise<MicrosoftGraph.PlannerPlan[]>;
```

Comment:
Update method names to the same format - for example, I'd change this to `getAllMyPlannerPlans`